### PR TITLE
document the module loading issue in the README.

### PR DESCRIPTION
**Pull request by jasonb-vbt**
Document the module loading issue in the README. (Issue #90)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,28 @@ There are many other options to explore. You can see them all by running:
...
 
+Submodule loading
+-----------------
+
+`pdoc` uses idiomatic Python when loading your modules. Therefore, for `pdoc`
+to find any submodules of the input module you specify on the command line,
+those modules must be available through Python's ordinary module loading process.
+
+This is not a problem for globally installed modules like `sys`, but can be
+a problem for your own sub-modules depending on how you have installed them.
```
Comment (BurntSushi) [inline README.md]: Could you add a clarification note that says `pdoc` works well with `virtualenv`? (Which is what "idiomatic Python" means to me.) Namely, if you're documenting your project, and you're using a `virtualenv`, then installing `pdoc` into that `virtualenv` and using it should work perfectly.

**Comment by BurntSushi** [comment]
@jasonb-vbt Thanks! I really appreciate the docs! I have one little nit but otherwise this looks good. :-)

File: README.md (hunk 1 above)
Comment (jasonb-vbt) [inline README.md]: Sure. To me that sounds like something to describe separately from the module loading issue, but I will add a quick note about that too. Hang on.

**Comment by BurntSushi** [comment]
Perfect. Thanks again!

**Comment by jasonb-vbt** [comment]
Fastest PR cycle ever. Thanks for being so responsive! :)

**Pull request merged**